.github/workflows/ci_tests_dev.yaml

```diff
@@ -76,7 +76,7 @@ jobs:
         uses: conda-incubator/setup-miniconda@v2.1.1
         with:
           activate-environment: pygmt
-          python-version: ${{ matrix.python-version }}
+          # python-version: ${{ matrix.python-version }}
```

Comment:
Any special reason to comment out this line?

Reply:
The `setup-miniconda` action won't work with Python 3.10, because there isn't a miniconda or mambaforge version with Python 3.10 yet (see https://repo.anaconda.com/miniconda/ and https://github.com/conda-forge/miniforge/releases). The Python 3.10 install step happens later below.